WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -433,22 +435,28 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String permissi
                 if (permissions == null || permissions.length == 0) {
```

Comment:
Minor nit, to prevent a lint warning: `permissions` will never be null here so we can skip the null check.